sycl/unittests/misc/DefaultContext.cpp

```diff
@@ -34,3 +49,21 @@ TEST(DefaultContextTest, DefaultContextTest) {
   ASSERT_EQ(Dev1.get_platform().ext_oneapi_get_default_context(),
             Dev2.get_platform().ext_oneapi_get_default_context());
 }
+
+TEST(DefaultContextTest, DefaultContextCanBeDisabled) {
+  set_env(EnableDefaultContextsName, "0");
+
```

Comment:
Shouldn't we reset config after setting this env var?

Reply:
I think I broke caching mechanism for SYCL config. I'll fix it.